src/compas/robots/model/robot.py

```diff
@@ -704,7 +766,7 @@ def compute_transformations(self, joint_state, link=None, parent_transformation=
 
         Parameters
         ----------
-        joint_state : dict
+        joint_state : dict or :class:`compas.robots.Configuration`
```

Comment:
totally minor detail, but I'd swap the order of these to make `Configuration` the perceived preferred option